andela/ah-kgl-avengers-frontend
# #165020172 Show tags for each article

### What does this PR do?
- Display article's tags for each relevant view
### Description of Task to be completed?
- User should be able to see articles' tags
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ft-show-article-tags-165020172`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/`
- On the landing page, each article shows its tags.
- When you click on one article you can read the tags.
### Any background context you want to provide?
- The article which has no associated tag will show `#notags`
### What are the relevant pivotal tracker stories?
[#165020172](https://www.pivotaltracker.com/n/projects/2321840/stories/165020172)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29593858/60045626-94c25c00-96c5-11e9-8c93-1dc35e6c3991.png)

### Questions:
